shared/studio/tabs/dataview/state/index.ts

```diff
@@ -267,9 +269,10 @@ export class DataInspector extends Model({
   sortBy: prop<SortBy | null>(null),
   fieldWidths: prop<number[]>(() => []),
   expandedInspectors: prop(() => objectMap<ExpandedInspector>()),
+  expandedRowMobile: prop<ExpandedInspector | null>(null),
 
   filter: prop<string>(""),
-  filterError: prop<string>(""),
+  errorFilter: prop<ErrorFilter | null>(null),
```

Comment:
Not really a issue, but why flip the words? To me `filterError` sounds more 'correct'.

Reply:
because I also save the filter : ) 
`filterError` is more like the error string, and `errorFilter` is more like filter type